### ISSUE TITLE
bug: Calling `removeLine()` to empty `Textfield` deforms the first line of the `Textfield`'s box

### ISSUE THREAD

**Issue by luiz00martins** [Pyroxenium/Basalt#33]
### Minimal Working Example

![image](https://user-images.githubusercontent.com/43142209/196757817-f3c8ec21-e378-461c-918c-a381d34ab907.png)

```lua
local filePath = "/basalt.lua"
if not(fs.exists(filePath))then
    shell.run("pastebin run ESs1mg7P packed true "..filePath:gsub(".lua", ""))
end
local basalt = require(filePath:gsub(".lua", ""))

local main = basalt.createFrame("mainFrame")

-- This is fine.
local textfield_0 = main:addTextfield('textfield_0')
	:setPosition(1, 1)
	:setSize(20, 3)
	-- two addLine
	:addLine('Value')
	:addLine('Another Value')
	-- no removeLine

-- This is fine.
local textfield_1 = main:addTextfield('textfield_1')
	:setPosition(1, 5)
	:setSize(20, 3)
	-- two addLine
	:addLine('Value')
	:addLine('Another Value')
	-- one removeLine
	:removeLine()

-- This gets deformed.
local textfield_2 = main:addTextfield('textfield_2')
	:setPosition(1, 9)
	:setSize(20, 3)
	-- two addLine
	:addLine('Value')
	:addLine('Another Value')
	-- two removeLine
	:removeLine()
	:removeLine()

basalt.autoUpdate()
```

As seen in the MWE above, `textfield_2` gets deformed by the second call to `removeLine()`.

### Expected behavior

The `Textfield` maintains its shape.

![image](https://user-images.githubusercontent.com/43142209/196759139-4ecba66e-b3f6-4786-b0fd-dc8dc8c1429d.png)

### Checklist

[X] I am running the latest version.

**Comment by NoryiE** [comment]
Thank you very much, i will work on that in the weekend!

**Comment by NoryiE** [comment]
Hey, i fixed this issue here: [f8232d6](https://github.com/Pyroxenium/Basalt/commit/f8232d6d5f8bcbe4cf0da4386410386958c82716) - right now its only available in the dev branch and will take a while to be in the release branch. Here's a shell command for downloading the most recent dev branch: 

```
wget run https://basalt.madefor.cc/install.lua packed basalt.lua dev
```
